package.json

```diff
@@ -684,17 +684,69 @@
         "command": "xml.restart.language.server",
         "title": "Restart XML Language Server",
         "category": "XML"
+      },
+      {
+        "command": "xml.refactor.surround.with.tags",
+        "title": "Surround with Tags (Wrap)",
+        "category": "XML"
+      },
+      {
+        "command": "xml.refactor.surround.with.comments",
+        "title": "Surround with Comments",
+        "category": "XML"
+      },
+      {
+        "command": "xml.refactor.surround.with.cdata",
+        "title": "Surround with CDATA",
+        "category": "XML"
       }
     ],
     "menus": {
       "commandPalette": [
         {
           "command": "xml.validation.current.file",
-          "when": "editorLangId == xml"
+          "when": "editorLangId in xml.supportedLanguageIds && XMLLSReady"
+        },
+        {
+          "command": "xml.validation.all.files",
+          "when": "XMLLSReady"
         },
         {
           "command": "xml.command.bind.grammar",
-          "when": "resourceFilename =~ /xml/ && editorIsOpen"
+          "when": "resourceFilename =~ /xml/ && editorIsOpen && XMLLSReady"
+        },
+        {
+          "command": "xml.restart.language.server",
```

Comment:
I still see restart in the command palette when I open a blank window before opening any XML files.

Reply:
I was running an out  of date version of vscode-xml it seems